### PR TITLE
Add Uche's activity attempt

### PR DESCRIPTION
**Pull request by osiau**

### CODE REVIEW

File: activities/activity_uo.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    gs.write_command(
+        "v.in.ascii",
+        flags="t",
+        input="-",
+        output=points,
+        separator="comma",
+        stdin="638432,220382\n638621,220607",
+        env=env,
+    )
+    env = None
```
Comment (petrasovaa) [inline activities/activity_uo.py]: Not sure why you have it there, this breaks the checks.

Comment (osiau) [reply activities/activity_uo.py]: Why I have what where?

Comment (petrasovaa) [reply activities/activity_uo.py]: env = None

Comment (osiau) [reply activities/activity_uo.py]: I fixed that, but I got it from the LCP example.

Comment (petrasovaa) [reply activities/activity_uo.py]: You need to pass the env variable into run_LCP, not put None there.

Comment (osiau) [reply activities/activity_uo.py]: Yes, it's fixed now. It's no longer causing problems.

File: activities/activity_uo.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    gs.write_command(
+        "v.in.ascii",
+        flags="t",
+        input="-",
+        output=points,
+        separator="comma",
+        stdin="638432,220382\n638621,220607",
+        env=env,
+    )
+    run_LCP(scanned_elev=elev_resampled, env=None, points=points)
```
Comment (petrasovaa) [inline activities/activity_uo.py]: This must be env=env

Comment (osiau) [reply activities/activity_uo.py]: Ah ok that fixed the other issue, thank you.

**Pull request closed without merging**